x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/selectors.ts

```diff
@@ -117,17 +121,32 @@ export const fullPolicy: (s: Immutable<PolicyDetailsState>) => PolicyConfig = cr
   }
 );
 
+/**
+ * Returns the full Endpoint Policy, with any paid/licensed features that are not
+ * currently allowed for the current license level reset to defaults.
+ */
+const policyStrippedUnlicensedFeatures: (s: PolicyDetailsState) => PolicyConfig = createSelector(
```

Comment:
There is a method above on line 33 named `getPolicyDataForUpdate` that is used during update. Does it have to be updated as well? I believe its the one we use to get the data to send on the API update.